### ISSUE TITLE
taskcluster - derived_taskgroup_costs tweak

### ISSUE THREAD

**Issue by nthomas-mozilla** [mozilla-services/lua_sandbox_extensions#493]
I'm consuming the `wall_clock_seconds` field of `taskclusteretl.derived_taskgroup_costs` and ran into a situation where the value is being inflated. If we abandon a build for a release then shipit will trigger a cancel all action, which is intended to kill off any tasks running. Unfortunately this appears in the same graph as the real release work, and we sometimes don't cancel a build until a few days later.

I'd like to exclude the cancel task with a change to the scheduled query:
```diff
--- derived_taskgroup_costs.txt	2019-12-20 17:09:37.000000000 +1300
+++ derived_taskgroup_costs-v2.txt	2019-12-20 17:10:13.000000000 +1300
@@ -25,16 +25,18 @@
     project),
   b AS (
   SELECT
     taskGroupId,
     MIN(started) AS started,
     MAX(resolved) AS resolved
   FROM
     taskclusteretl.derived_task_summary dts
+  WHERE
+    symbol != "cAll"  -- exclude cancel-all tasks when a release build is abandoned in ship-it
   GROUP BY
     taskGroupId)
 SELECT
   a.*,
   b.started,
   b.resolved,
   TIMESTAMP_DIFF(b.resolved, b.started, millisecond) / 1000 AS wall_clock_seconds,
   release_promotion_flavor,
```
I don't think this will affect anyone else using `wall_clock_seconds` because cancel-all is usually used on try or autoland when there are active tasks, and they'll finish last. It's just a problem for releases when we respin after a few days (eg RC week).

Could we also re-calculate `wall_clock_seconds`, for any rows of `derived_taskgroup_costs` with non-null value for `release_promotion_flavor` ?

(cc @srfraser)

**Comment by nthomas-mozilla** [comment]
Simon and I looked at this a bit today. The where condition actually needs to be 
```symbol is null or symbol != "cAll"```
because the original comparison excluded tasks with a null symbol. With that we created a temp table and the dashboards where much improved. 

TODO
* get the change into github
* modify the scheduled query
* regenerate the `derived_taskgroup_costs` using a tweaked version new query
